src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -35,6 +36,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
   //the dispatcher is visible for all async operations in the following code
   implicit val system: ActorSystem = ActorSystem("lorre-system")
   implicit val dispatcher = system.dispatcher
+  implicit val nodeContext = RemoteContext(tezosConf, callsConf, streamingClientConf)
```

Comment:
Minor nitpick, maybe callsConf should be renamed networkTimeoutConf, and the class associated with it should be renamed as such? It seems the config is only used to update that aspect of network calls.

Reply:
I'll look into that, the final version of this whole re-work is quite different, might be that I've no more need of those.

No worry though, I'll incorporate your feedbacks in the latest proposal for this work, on which I hope to spend some time soon, it's almost ready.

Reply:
Ok, I've verified and what you suggested makes perfect sense, I renamed the config in the end result, I'll publish in a separate PR